querybook/webapp/components/DataTableViewSourceQuery/DataTableViewSourceQuery.tsx

```diff
@@ -80,6 +81,21 @@ export const DataTableViewSourceQuery: React.FunctionComponent<IProps> = ({
         );
     });
 
+    const customProperties = table.custom_properties ?? {};
+    const workflowValue = customProperties['workflow']?.toString() ?? null;
```

Comment:
nit: the workflow is already a string I think?

Reply:
custom_property values are string | number, so change to string type for Link component

Reply:
@zhangvi7 it doesn't make sense to convert if it is a number

Reply:
updated to use typeof to assert string type